### ISSUE TITLE
Latest update added new bug

### ISSUE THREAD

**Issue by TheLightBringer1991** [wiremod/advdupe2#198]
Stuff was working better before the change, I think it whould be rolled back to the previous version. Tested my dupes on servers that have the previous version and everything works fine, but instantly breakes latches with the new version of the code.

**Comment by thegrb93** [comment]
Ok, I found another improvement in the sorting, basically it wasn't taking into account the limit being hit.

**Comment by thegrb93** [comment]
This should fix any remaining issues with limit hitting. https://github.com/wiremod/advdupe2/pull/200

As for wire latch. That has always had the same issue and won't be fixed until https://github.com/wiremod/wire/issues/1830 and https://github.com/Facepunch/garrysmod/pull/1556 are fixed

**Comment by TheLightBringer1991** [comment]
Fixes nothing, actually adds an additional error where it can't create a nocollide constraint.

**Comment by TheLightBringer1991** [comment]
[5t CATZ537 8x8 B.txt](https://github.com/wiremod/advdupe2/files/2776343/5t.CATZ537.8x8.B.txt)

Does it with all dupes, try it with this one and see the constraint count go up everytime you dupe it and spawn it.

**Comment by TheLightBringer1991** [comment]
Now I'm going to bed, will check it as soon as I can.

**Comment by TheLightBringer1991** [comment]
Works a bit better, but still constraints feel looser a bit after duping, even if nothing was changed. I spawn a tank, dupe it as is, spawn it again, latches go a  bit loose.

**Comment by thegrb93** [comment]
Yeah the latch is connecting two props in a full constraint system.

**Comment by thegrb93** [comment]
won't be fixed until wiremod/wire#1830 and Facepunch/garrysmod#1556 are fixed

**Comment by TheLightBringer1991** [comment]
That would be good to test at least.
